client/my-sites/plugins/plan-setup/index.jsx

```diff
@@ -1,81 +1,336 @@
 /**
```

Comment:
Got it. Whatever makes the most sense to you, just having the jetpack prefix fixes the main problem.